### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by SunnyIMG** [DrUm78/FunKey-OS#4]
FunKey-OS/FunKey/package/sdl/sdl.mk:95: *** Package 'sdl' defined a second time in '/home/FunKey-OS/FunKey/package/sdl/'; previous definition was in 'package/sdl/'.  Stop.
make: *** [Makefile:23: _all] Error 2

**Comment by DrUm78** [comment]
Yeah that error is normal. For now the new OS cannot be compiled without a few local modifications that are not committed yet on release branch because they impact buildroot submodule.
Also don't use drum78 branch as it's only for my personal need, it's not for regular users.
